jobs/cache_maintenance/src/cache_maintenance/config.py

```diff
@@ -30,29 +30,40 @@ def from_env(cls) -> "BackfillConfig":
 
 
 DUCKDB_INDEX_CACHE_DIRECTORY = None
-DUCKDB_INDEX_SUBDIRECTORY = "downloads"
-DUCKDB_INDEX_EXPIRED_TIME_INTERVAL_SECONDS = 10 * 60  # 10 minutes
-DUCKDB_INDEX_FILE_EXTENSION = ".duckdb"
+DUCKDB_INDEX_DOWNLOADS_SUBDIRECTORY = "downloads"
+DUCKDB_INDEX_JOB_RUNNER_SUBDIRECTORY = "job_runner"
+DUCKDB_INDEX_DOWNLOADS_EXPIRED_TIME_INTERVAL_SECONDS = 10 * 60  # 10 minutes
+DUCKDB_INDEX_JOB_RUNNER_EXPIRED_TIME_INTERVAL_SECONDS = 10 * 60  # 10 minutes
 
 
 @dataclass(frozen=True)
 class DuckDbConfig:
     cache_directory: Optional[str] = DUCKDB_INDEX_CACHE_DIRECTORY
-    subdirectory: str = DUCKDB_INDEX_SUBDIRECTORY
-    expired_time_interval_seconds: int = DUCKDB_INDEX_EXPIRED_TIME_INTERVAL_SECONDS
-    file_extension: str = DUCKDB_INDEX_FILE_EXTENSION
+    downloads_subdirectory: str = DUCKDB_INDEX_DOWNLOADS_SUBDIRECTORY
+    downloads_expired_time_interval_seconds: int = DUCKDB_INDEX_DOWNLOADS_EXPIRED_TIME_INTERVAL_SECONDS
+    job_runner_subdirectory: str = DUCKDB_INDEX_JOB_RUNNER_SUBDIRECTORY
+    job_runner_expired_time_interval_seconds: int = DUCKDB_INDEX_JOB_RUNNER_EXPIRED_TIME_INTERVAL_SECONDS
 
     @classmethod
     def from_env(cls) -> "DuckDbConfig":
         env = Env(expand_vars=True)
         with env.prefixed("DUCKDB_INDEX_"):
             return cls(
                 cache_directory=env.str(name="CACHE_DIRECTORY", default=DUCKDB_INDEX_CACHE_DIRECTORY),
-                subdirectory=env.str(name="SUBDIRECTORY", default=DUCKDB_INDEX_SUBDIRECTORY),
-                expired_time_interval_seconds=env.int(
-                    name="EXPIRED_TIME_INTERVAL_SECONDS", default=DUCKDB_INDEX_EXPIRED_TIME_INTERVAL_SECONDS
+                downloads_subdirectory=env.str(
```

Comment:
maybe it's simpler and easier to extend to just have one `EXPIRED_TIME_INTERVAL_SECONDS` instead of `DUCKDB_INDEX_DOWNLOADS_EXPIRED_TIME_INTERVAL_SECONDS`, `DUCKDB_INDEX_JOB_RUNNER_EXPIRED_TIME_INTERVAL_SECONDS` and `DATASETS_BASED_EXPIRED_TIME_INTERVAL_SECONDS`. Same for the directory.

And the two values are set as environment variables (have only one action: clean-directory)?

Feel free to ignore if it's too much change

Reply:
imo it's nice to have the prefixes

Reply:
OK. Let's ignore the comment

Reply:
I thought it was better to factorize the parameters so that we can use them for other folders quickly (Maybe for statistics?).
Maybe have a look at the last commit? Let me know if I have to rollback.

Reply:
thanks for refactoring to have only one action, it's exactly what I had in mind